packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionIntegration.tests.js

```diff
@@ -641,5 +644,36 @@ export default function() {
             this.$element.find('p').first().text('@');
             this.clock.tick(10);
         });
+
+        test('aria-activedescendant should exist on textbox when mention pops up', function(assert) {
+            const done = assert.async();
+            const valueChangeSpy = sinon.spy(() => {
+                const $content = this.$element.find(`.${HTML_EDITOR_CONTENT}`);
```

Comment:
This seems to be excess cause `$content` specified in 672th row should be accessible here because of a closure